src/code/z_player_lib.c

```diff
@@ -1687,9 +1687,8 @@ PlayerExplosive Player_GetExplosiveHeld(Player* player) {
 PlayerSword Player_SwordFromIA(Player* player, PlayerItemAction itemAction) {
     PlayerSword sword = PLAYER_SWORD_KOKIRI;
 
-    //! FAKE:
     if ((itemAction == PLAYER_IA_LAST_USED) ||
```

Comment:
Note I just removed the fake label here as I actually don't think this is a fake.